demo/map.js

```diff
@@ -376,4 +380,31 @@ function showLegend() {
   }
 
   document.body.appendChild(legendBlock);
-}
\ No newline at end of file
+};
+
+function showAddress(e) {
```

Comment:
Čia gal išėitu ir nekūrti naują funkciją, o gal geriau pabandyti panaudoti mano sukurta funkciją `poiOnClick`.
Turėjau minčių perdaryti `interactiveLayerId` šį kintamajį į masyvą, kad būtų ne vienas sluoksnis interaktyvus, o keli.
Padarius šitą pakeitimą, nereikėtų tavo funkcijos. Adresą aš jau kaip ir formuoju, tad būtų tik galima pakeisti tik į tavo formatą.

Reply:
Pritariu, bet refactorinti geriau tada, kai matosi visi atvejai ir niuansai. Kolkas tai tik eksperimentas. Su laiku sudėliosim gražiai ;)